test/test_mps.py

```diff
@@ -4077,7 +4100,7 @@ def test_no_warning_on_import(self):
     def _get_not_implemented_op(self):
         # This can be changed once we actually implement `torch.eye`
         # Should return fn, args, kwargs, string_version
-        return torch.eye, (2,), {"device": "mps"}, "torch.eye(2, device='mps')"
+        return torch.bincount, (torch.tensor([4, 3, 6, 3, 4], device='mps')), {}, "torch.bincount(torch.tensor([4, 3, 6, 3, 4], device='mps'))"
```

Comment:
nit: update the comment above to say `bincount`.

Reply:
Done.